packages/wrangler/src/__tests__/metrics.test.ts

```diff
@@ -17,18 +18,17 @@ import { clearDialogs, mockConfirm } from "./helpers/mock-dialogs";
 import { useMockIsTTY } from "./helpers/mock-istty";
 import { msw, mswSuccessOauthHandlers } from "./helpers/msw";
 import { runInTempDir } from "./helpers/run-in-tmp";
+import type { MockInstance } from "vitest";
 
 declare const global: { SPARROW_SOURCE_KEY: string | undefined };
+vi.unmock("../metrics/metrics-config");
```

Comment:
This replaces the use of a module level `useOriginal` flag, which wasn't working properly in Vitest

Reply:
I thought the main goal of moving from jest to vitest was to unravel the global mocking and unmocking and avoid the pattern – do we have a plan to do this in a follow-up?

Reply:
One of the issues with Jest was mocking modules globally and then unmocking them with a module level `useOriginal` flag (rather than calling `jest.unmock()`). Using `vi.unmock()` here makes it clearer what's going on, I think. 

We should definitely improve this in a followup though (and make sure to only mock metrics-config when needed). I kinda started that in this PR for some things (`ws` is no longer mocked globally, for instance, it's only mocked in the two files that need it), but I didn't want to make this a refactor as well as a migration.